jobs/pending_payments.ts

```diff
@@ -145,9 +149,10 @@ exports.attemptCommunitiesPendingPayments = async (bot: Telegraf<MainContext>):
       }
 
       const community = await Community.findById(pending.community_id);
+      if (community === null) return;
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program